base/setup/usetup/usetup.c

```diff
@@ -626,6 +626,174 @@ UpdateKBLayout(VOID)
     }
 }
 
+/*
+ * Function Name: ProgressTimeOutStringHandler
+ *
+ * Purpose: Handles the generation (displaying) of the timeout countdown to the screen
+ *			dinamically
+ *
+ * Return: FALSE
+ */
+static
+BOOLEAN NTAPI
+ProgressTimeOutStringHandler(
+	IN PPROGRESSBAR Bar,
+	IN BOOLEAN AlwaysUpdate,
+	OUT PSTR Buffer,
+	IN SIZE_T cchBufferSize)
+{
+	ULONG OldProgress = Bar->Progress;
```

Comment:
I see tabs here instead of 4 spaces, is it intentionally?

Reply:
Looks like VS 2017 saved the lines in tabs albeit the whole file being saved in using spaces as indentation. I'll fix this.

Reply:
The default settings of the IDE can be changed in its Personalization dialog.